scripts/py_matter_yamltests/matter_yamltests/definitions.py

```diff
@@ -141,11 +141,19 @@ def get_type_by_name(self, cluster_name: str, target_name: str):
         if struct:
             return struct
 
+        event = self.get_event_by_name(cluster_name, target_name)
+        if event:
+            return event
+
         return None
 
     def is_fabric_scoped(self, target) -> bool:
-        if hasattr(target, 'qualities'):
+        if isinstance(target, Event):
+            return target.is_fabric_sensitive
```

Comment:
I think this need to be `return bool(return target.is_fabric_sensitive)`. is_fabric_sensitive returns an enum